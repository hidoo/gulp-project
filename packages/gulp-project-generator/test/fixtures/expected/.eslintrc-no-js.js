module.exports = {
  'root': true,
  'extends': [
    '@hidoo/eslint-config',
    '@hidoo/eslint-config/+babel'
  ],
  'overrides': [
    // for Node
    {
      'files': [
        'src/server/**/*.js',
        'task/**/*.js',
        'gulpfile.babel.js'
      ],
      'extends': [
        '@hidoo/eslint-config/+node'
      ]
    }
  ]
};
