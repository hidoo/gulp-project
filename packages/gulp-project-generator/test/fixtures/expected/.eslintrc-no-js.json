{
  "$schema": "https://json.schemastore.org/eslintrc.json",
  "root": true,
  "extends": [
    "@hidoo/eslint-config",
    "@hidoo/eslint-config/+babel"
  ],
  "overrides": [
    {
      "files": [
        "src/server/**/*.js",
        "task/**/*.js",
        "gulpfile.js"
      ],
      "extends": [
        "@hidoo/eslint-config/+node"
      ],
      "rules": {
        "node/file-extension-in-import": [
          "error",
          "always"
        ]
      }
    }
  ]
}
